library/src/morpheus-master/sass/modules/tool/authz/_authz.scss

```diff
@@ -48,19 +49,26 @@
   display: block;
 }
 .checkGrid tr.evenrow .permissionDescription {
-  background: #f4f9fe;
+  background: var(--sakai-background-color-1);
 }
 .checkGrid tr.evenrow td {
-  background: #f7fbff;
+  background: var(--sakai-background-color-1);
 }
 .checkGrid .rowHover {
-  background: #c4dffb !important;
+  background: var(--sakai-background-color-2);
```

Comment:
Have you confirmed that these dropped !importants are no longer needed in this file?

Reply:
I was thinking that css props would reduce need of !important but that's not the case, good find.